docs/testing/debug-office-add-ins-on-ipad-and-mac.md

```diff
@@ -28,6 +28,12 @@ Then, open the Office application and insert your add-in. Right-click the add-in
 
 > [!NOTE]
 > Please note that this is an experimental feature and there are no guarantees that we will preserve this functionality in future versions of Office applications.
+>
+> If you are trying to use the inspector, and the dialog is flickering, try the following workaround:
+> 1. Resize the dialog such that it's a very small window.
```

Comment:
Is "very small" descriptive enough? Do users need to make the window as small as possible? Or just smaller than the new window?

Reply:
Updated